Cargo.toml

```diff
@@ -28,7 +28,8 @@ signal-hook = "0.1.16"
 structopt = "0.3.15"
 
 [features]
-defmt = ["defmt-elf2table", "defmt-decoder"]
+# this feature does nothing and it's kept for backwards compatibility; it will be removed in the next semver bump
```

Comment:
nit: would it make sense to note this down as a low-prio issue as a reminder or is that overkill?

Reply:
hmm, actually I think adding a TODO comment to this Cargo.toml just above the `version = ` field would be more visible. (That way whoever bumps the minor version next will be reminded to remove this feature)

Reply:
ha! yeah that makes sense and causes less clutter